Cython/Includes/libcpp/vector.pxd

```diff
@@ -73,9 +75,11 @@ cdef extern from "<vector>" namespace "std" nogil:
         void pop_back()
         void push_back(T&) except +
         reverse_iterator rbegin()
-        const_reverse_iterator const_rbegin "crbegin"()
```

Comment:
This was most likely a mistake, since in all other cases the `const_*` version of the api is used to explicitly call the const version of `rbegin`. Even though now that `iterators` can be properly converted to `const_iterators` (https://github.com/cython/cython/pull/4528) these custom functions are not really required anymore.

Reply:
> Even though now that iterators can be properly converted to const_iterators (#4528) these custom functions are not really required anymore.

Is this a manual conversion or did you find a way to make it work automatically?

Reply:
It works automatically, since I provided `const_iterator` with an assignment operator for `iterator`
```
cppclass const_iterator:
    const_iterator() except +
    const_iterator(iterator&) except +
    const_iterator(const_iterator&) except +
    operator=(iterator&) except +
```

so it is possible to use
```
cdef test(const vector[int]& x):
    cdef vector[int].const_iterator iter = x.begin()
```
or for c++11 it would be possible to use

```
cdef test(const vector[int]& x):
    iter = x.cbegin()
```
however
```
cdef test(const vector[int]& x):
    iter = x.begin()
```
would lead to broken code, since cython at least right now can not differentiate between const and non const. It would be optimal if it would be possible to specify something like:
```
iterator begin()
const_iterator() const
```
where cython uses the const version if:
- the class is const
- no fitting non const function exists